### PR TITLE
Set Mayflower default pattern to greeting / readme page

### PR DESCRIPTION
**Pull request by jesconstantine**
This PR creates a (sparse) homepage or Mayflower, which currently displays the version, date, and a greeting.

This page should be more of a readme than a greeting, but this is a start. (Please feel free to add more information, suggest better patterns, plan for the future, etc.)

Would love to add to the deploy process a step where the version gets bumped and the contents of this page are updated.

### Screenshot of current page content
image

### CODE REVIEW

**Comment by gender404** [comment]
Looks good to me.  I added the green diagonal background to this.  

image

File: styleguide/source/_patterns/05-pages/Readme.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "narrowTemplate": {
+    "side": "left",
+    "color": "green"
+  },
+
+  "errorPage": {
+    "type": "v2.2.1 (01/09/2016)",
```
Comment (isaacchansky) [inline styleguide/source/_patterns/05-pages/Readme.json:8]: Is this the only place we update the Version number? I'm thinking about keeping it in sync if it's in more than one spot...

**Review by isaacchansky — COMMENTED** [review]
Looks good! nice work. I'll defer to Harlan, but I could see us maybe wanting to drop the background color if this gets more text-heavy, for readability.

**Review by gender404 — APPROVED** [review]
If we get more content to put in here, we might want to get the design team involved.  For now, I think this is a good start.

File: styleguide/source/_patterns/05-pages/Readme.json (hunk 1 above)
Comment (jesconstantine) [reply styleguide/source/_patterns/05-pages/Readme.json:8]: It's the only place to my knowledge.  @legostud - can you think of any other place?

Comment (gender404) [reply styleguide/source/_patterns/05-pages/Readme.json:8]: the release-notes.md file in the repo is the only other place I can think of.

Comment (jesconstantine) [reply styleguide/source/_patterns/05-pages/Readme.json:8]: Oh, yes, right.  That makes sense. :)

At any rate, @isaacchansky this speaks to my desire to automate.  But I thought this was rather MVP...

Comment (isaacchansky) [reply styleguide/source/_patterns/05-pages/Readme.json:8]: yep! totally agree, I like it as is.

**Comment by gender404** [comment]
this ticket seems quite now so I'm assuming it's fine to merge this request even though the reviewers haven't all submitted an approval.

**Comment by jesconstantine** [comment]
I agree: So I'm game if either @legostud or @isaacchansky want to merge.

**Pull request merged**